internal/configure/topology/dc_item.go

```diff
@@ -32,7 +32,7 @@ const (
 	REQUIRE_STRING
 	REQUIRE_BOOL
 	REQUIRE_POSITIVE_INTEGER
-	REQUIRE_SLICE
+	REQUIRE_STRING_SLICE
```

Comment:
Should `const REQUIRE_STRING_SLICE` in `dc_item.go` be removed since `const REQUIRE_STRING_SLICE` in `item_set.go` covers most cases ?

Reply:
Yeah, it is unnecessary.